datadog.tf

```diff
@@ -0,0 +1,133 @@
+locals {
+  datadog_operator_set = merge({
+    "site"                      = "datadoghq.eu",
+    "resources.requests.cpu"    = "10m",
+    "resources.requests.memory" = "50Mi",
+  }, var.datadog_operator_values)
+
+  datadog_agent_manifest_features = try(indent(2, join("", ["\n", var.datadog.agent_manifest_features])), indent(2, <<-YAML
+
```

Comment:
This empty line is needed because `indend()` doesn't indent the first line. This is also the reason for the join at line 8. Same applies for line 19

Reply:
agreed, removed as discussed